chaco/tests/test_jitterplot.py

```diff
@@ -8,10 +8,10 @@
 from chaco.api import (
     ArrayDataSource,
     DataRange1D,
+    JitterPlot,
     LinearMapper,
     PlotGraphicsContext,
 )
-from chaco.jitterplot import JitterPlot
```

Comment:
I realized this is a common practice in chaco:
in a test module for ____ importing everything except ___ from `chaco.api` and then ___ from the source.  I do not know why this is (hence me undoing that here and just importing `JitterPlot` from the api module).

If it is purposeful I am happy to undo this change.  If it is not (i.e maybe test was written before the class was available in the api module?) I will leave it, and make the similar changes elsewhere in the code base.